lib/modules/platform/gerrit/index.spec.ts

```diff
@@ -247,7 +278,11 @@ describe('modules/platform/gerrit/index', () => {
     });
   });
 
-  describe('createPr() - error', () => {
+  describe('createPr()', () => {
+    beforeEach(() => {
+      vi.setSystemTime(t0.toMillis());
```

Comment:
set it in before all where you set fake timers

Reply:
I changed it, but that only works because the tests don't take 5 minutes to execute.